<launch>
    <arg name="arm_cam_name" value="arm_cam" />
    <env name="GSCAM_CONFIG" value="v4l2src device=/dev/video0 ! video/x-raw-yuv,framerate=15/1,width=320,height=240 ! ffmpegcolorspace " />
    <node pkg="gscam" type="gscam" name="$(arg arm_cam_name)">
        <param name="camera_name" value="$(arg arm_cam_name)" />
        <remap from="camera/image_raw" to="$(arg arm_cam_name)/image_raw" />
    </node>
    <!--  <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" respawn="false" output="screen">
         <param name="autosize" value="true" />
        </node> -->
    <node pkg="ball_detect" name="arm_ball_detect" type="arm_ball_detect.py" output="screen" />
</launch>